backend-rust/src/migrations.rs

```diff
@@ -171,14 +171,16 @@ pub enum SchemaVersion {
          performance."
     )]
     IndexBlocksWithNoCumulativeFinTime,
+    #[display("0003:PayDayPoolCommissionRates")]
+    PayDayPoolCommissionRates,
 }
 impl SchemaVersion {
     /// The minimum supported database schema version for the API.
     /// Fails at startup if any breaking database schema versions have been
     /// introduced since this version.
     pub const API_SUPPORTED_SCHEMA_VERSION: SchemaVersion = SchemaVersion::InitialFirstHalf;
```

Comment:
Since the API now rely on the new information in the database schema we need to bump this version:
```suggestion
    pub const API_SUPPORTED_SCHEMA_VERSION: SchemaVersion = SchemaVersion::PayDayPoolCommissionRates;
```